src/Header.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flexCenter, flexBetween } from './css/mixins';
+
+const Header = () => {
+  return (
+    <StyledHeader>
+      <FlexBetween>
+        <Title>Ordering</Title>
+        <Gnb>
+          <GnbTap>
+            <GnbTitle>든든한 메인요리</GnbTitle>
+            <GnbCategory>
+              <GnbContent>육류 요리</GnbContent>
+              <GnbContent>해산물 요리</GnbContent>
+            </GnbCategory>
+          </GnbTap>
+
+          <GnbTap>
+            <GnbTitle>뜨끈한 국물요리</GnbTitle>
+            <GnbCategory>
+              <GnbContent>국/탕/찌개</GnbContent>
+            </GnbCategory>
+          </GnbTap>
+
+          <GnbTap>
+            <GnbTitle>정갈한 밑반찬</GnbTitle>
+            <GnbCategory>
+              <GnbContent>나물/무침</GnbContent>
+              <GnbContent>조림/볶음</GnbContent>
+              <GnbContent>절임/장아찌</GnbContent>
+            </GnbCategory>
+          </GnbTap>
+        </Gnb>
+      </FlexBetween>
+
+      <Menu>
+        <MenuIcon src="assets/search.png" alt="search"></MenuIcon>
```

Comment:
```suggestion
        <MenuIcon src="assets/search.png" alt="search" />
```
children이 없으면 self closing으로 줄여서 적어주시면 더 좋을 것 같네요